main.js

```diff
@@ -1,3 +1,44 @@
 import './style.css';
 
-console.log('Hello world');
+document.querySelector(".button2").addEventListener("click", function () {
+    helptip(this);
+});
+
+function helptip(t) {
+    helptip.v = 0;
```

Comment:
Also, v and b variables are quite hard to read. it is ok to write veryLongAndValuableName for variables.